modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.tsx

```diff
@@ -213,56 +225,23 @@ function ActionsDropdown({
 		return <ClayLoadingIndicator className="mb-2 mt-2" />;
 	}
 
-	const renderItems = (items: IItemsActions[]) =>
-		items.map(({items: nestedItems = [], separator, type, ...item}, i) => {
-			if (type === 'group') {
-				if (!nestedItems.length) {
-					return;
-				}
-
-				return (
-					<ClayDropDown.Group {...item} key={i}>
-						{separator && i !== 0 && <ClayDropDown.Divider />}
-
-						{renderItems(nestedItems)}
-					</ClayDropDown.Group>
-				);
-			}
-
-			return (
-				<DropdownItem
-					action={item}
-					className={item.className}
-					closeMenu={() =>
-						onMenuActiveChange && onMenuActiveChange(false)
-					}
-					key={i}
-					onClick={onClick}
-					setLoading={setLoading}
-					url={
-						item.href &&
-						formatActionURL(item.href, itemData, item.target)
-					}
-				/>
-			);
-		});
-
 	return (
 		<div className="d-flex">
 			{inlineEditingAlwaysOn && inlineEditingActions}
 
-			<ClayDropDown
+			<ClayDropDownWithItems
 				active={menuActive}
-				hasLeftSymbols={hasIcons}
+				items={items}
+				menuElementAttrs={{onClick: (event) => event.stopPropagation()}}
 				onActiveChange={() =>
 					onMenuActiveChange && onMenuActiveChange(!menuActive)
 				}
-				onClick={(event) => event.stopPropagation()}
 				trigger={
 					<ClayButton
 						className="component-action dropdown-toggle"
 						disabled={loading}
 						displayType="unstyled"
+						onClick={(event) => event.stopPropagation()}
```

Comment:
These are needed to prevent the row selection to toggle.

Previously this was on <ClayDropDown onClick={(event) => event.stopPropagation()} /> but `onClick` isn't a valid prop so I added it here and using `menuElementAttrs`.